src/draw/vg_lite/lv_draw_vg_lite_vector.c

```diff
@@ -118,6 +115,29 @@ static void task_draw_cb(void * ctx, const lv_vector_path_t * path, const lv_vec
     float min_x, min_y, max_x, max_y;
     lv_vg_lite_path_get_bonding_box(lv_vg_path, &min_x, &min_y, &max_x, &max_y);
 
+    if(vg_lite_query_feature(gcFEATURE_BIT_VG_SCISSOR)) {
+        /* set scissor area */
+        lv_vg_lite_set_scissor_area(&dsc->scissor_area);
+    }
+    else {
+        /* calc inverse matrix */
+        vg_lite_matrix_t result;
+        if(!lv_vg_lite_matrix_inverse(&result, &matrix)) {
+            LV_LOG_ERROR("no inverse matrix");
+            LV_PROFILER_END;
+            return;
+        }
+
+        /* Reverse the clip area on the source */
+        lv_point_precise_t p1 = { dsc->scissor_area.x1, dsc->scissor_area.y1 };
+        lv_point_precise_t p1_res = lv_vg_lite_matrix_transform_point(&result, &p1);
+
+        lv_point_precise_t p2 = { dsc->scissor_area.x2, dsc->scissor_area.y2 };
+        lv_point_precise_t p2_res = lv_vg_lite_matrix_transform_point(&result, &p2);
+
+        lv_vg_lite_path_set_bonding_box(lv_vg_path, p1_res.x, p1_res.y, p2_res.x, p2_res.y);
```

Comment:
I tested and it may happen +-1 pixel error. Should we consider add +1 pixel here?

Reply:
fixed